modules/samples/artifacts/PublishJmsInXmlFormat/PublishJmsInXmlFormat.siddhi

```diff
@@ -7,13 +7,23 @@ Purpose:
 
 Prerequisites:
     1) Setup ActiveMQ
-        * Download activemq-client-5.x.x.jar (http://central.maven.org/maven2/org/apache/activemq/activemq-client/5.9.0/activemq-client-5.9.0.jar)
-        * Download apache-activemq-5.x.x-bin.zip (http://archive.apache.org/dist/activemq/apache-activemq/5.9.0/apache-activemq-5.9.0-bin.zip)
+        * Download activemq-client-5.x.x.jar (https://repo1.maven.org/maven2/org/apache/activemq/activemq-client/5.9.0/activemq-client-5.9.0.jar)
+        * Copy that jar into the {WSO2SIHome}/samples/sample-clients/lib directory.
         * ActiveMQ activemq-client-5.x.x.jar lib to be added and converted to OSGI (See Note: To convert ActiveMQ lib to OSGI).
-        * Unzip the apache-activemq-5.x.x-bin.zip and copy the following ActiveMQ libs in apache-activemq-5.x.x/lib to {WSO2SIHome}/samples/sample-clients/lib and {WSO2SIHome}/lib
-              - hawtbuf-1.9.jar
-              - geronimo-j2ee-management_1.1_spec-1.0.1.jar
-              - geronimo-jms_1.1_spec-1.1.1.jar
+        * Add following jars into the {WSO2SIHome}/samples/sample-clients/lib and {WSO2SIHome}/lib,
```

Comment:
isnt this common for both approaches ?

Reply:
When manually installing, user has to copy those 3 jars into the lib and samle lib directories, other than that other steps are similar to both approaches

Reply:
after setup activemq shall we start with This can be achieved in two ways, no need to mention what are the stuff goinfg to do in the begining